### ISSUE TITLE
GA tracking of video plays

### ISSUE THREAD

**Issue by joehobson** [navnorth/wp-oet-theme#22]
They would like to see tracking of video plays on Stories like we are doing on GAT. Look at the tracking code and see if you can implement it for Stories that have a YouTube video. There are other video embeds on the site as well so if we can have those tracked without having to do too much work when embedding videos that would be great too. 

We will probably also want to update the event labels so we can tell the difference between a video playback on GAT versus on Story versus just in page content. 

This is lower priority than other tasks, but since we're sending a theme update to the host, would be nice to get this update in as well if we can.

![screen shot 2015-07-09 at 9 47 23 am](https://cloud.githubusercontent.com/assets/838671/8598325/ab705ab2-261f-11e5-9f99-63bdd54c0e06.png)

**Comment by johnpaulbalagolan** [comment]
Finally fixed #22 with the latest source code in github I committed. There are two things need to remember though when using featured_video shortcode. Youtube embed url should have the enablejsapi set to 1. For multiple videos in a page, specify different ids on the featured_video short code. Here's an example of shortcode that works on GA Video Tracking

[featured_video src="//www.youtube.com/embed/tC36GKeFe98?enablejsapi=1" height="400" description="Superintendents discuss how and why they are connected, and why they believe their students must be connected during the May 2014 technical working group." id="ytplayer2"]

**Comment by joehobson** [comment]
Based on this, maybe we should switch the shortcode to use youTubeID instead of URL, just to be sure it plays right. All of their videos are YouTube, and I don't know that new people would put the /embed/ version of the URL in consistently. If we just get ID and then we put the YouTube embed code in ourselves, with the enablejsapi param, then we're sure tracking works. What do you think?

**Comment by johnpaulbalagolan** [comment]
Yeah, I think that will work sir. I will make the adjustment accordingly. 

Thanks.

**Comment by johnpaulbalagolan** [comment]
Hi Sir,

I have updated the featured_video shortcode to use youtube id to display video instead of using the youtube url. In this way, the enablejsapi youtube parameter is always set as the video embed code is generated dynamically. The updated source code is already committed to github.

Fyi. Thanks.
